includes/defaults.inc.php

```diff
@@ -705,6 +705,7 @@
 $config['poller_modules']['cisco-otv']                   = 1;
 $config['poller_modules']['ntp']                         = 1;
 $config['poller_modules']['services']                    = 1;
+$config['poller_modules']['loadbalancers']               = 1;
```

Comment:
We now have the ability to enable / disable these on a per os basis so set them to 0 here and enable them in the os. See `$os = 'linux';` in definitions.inc.php to see.